AppServices/Validators/CustomerValidations.cs

```diff
@@ -0,0 +1,98 @@
+﻿using FluentValidation;
+using System.Linq;
+
+namespace CustomerRegister.AppServices.Validators
+{
+    public class CustomerValidations : AbstractValidator<CustomerEntity>
+    {
+        public CustomerValidations()
+        {
+            RuleFor(x => x.FullName).
+                NotEmpty();
+
+            RuleFor(x => x.Email).
+                NotEmpty().
+                MinimumLength(4).
+                EmailAddress().Equal(x => x.EmailConfirmation);
+
+            RuleFor(x => x.Cpf).
+                NotEmpty().
+                Must(IsValidCpf).WithMessage("Por favor insira um CPF válido, este CPF está incorreto");
```

Comment:
Eu mudaria a mensagem para: "CPF incorreto. Por favor insira um CPF válido no formato 000.000.000-00". Porque dai você tenta dar uma direção ao usuário de como deve ser a formatação ao menos, apesar de que quando o erro for em estar com um CPF inválido, por exemplo 111.111.111-11 não tem muito como explicar mesmo, além de dizer que tá invalido.